examples/with-vite-react/package.json

```diff
@@ -21,7 +21,8 @@
     "@web3-onboard/mew": "^2.1.3",
     "@web3-onboard/portis": "^2.1.3",
     "@web3-onboard/react": "^2.3.5",
-    "@web3-onboard/sequence": "^2.0.2",
+    "@web3-onboard/sequence": "^2.0.3",
+    "@0xsequence": "^0.42.8",
```

Comment:
Are these changes required in the example projects?

Reply:
The `@web3-onboard/sequence` is updated to match the updated version of the connector from this PR.

Because of the version update above, the `@0xsequence` dependency becomes necessary  since `@web3-onboard/sequence` includes it as a `peerDependency`, meaning it needs to be installed together with the connector.

Reply:
@SamueleA right but once the new sequence version is released this import will not be required as the sequence package will pull in the dependency correct?

Reply:
That's correct! My bad!

I removed the 0xSequence dependency

Reply:
@SamueleA great! Once you have that pushed I can test it out, thanks!

Reply:
```suggestion
```